planning/obstacle_stop_planner/src/node.cpp

```diff
@@ -210,7 +210,7 @@ void ObstacleStopPlannerNode::onPointCloud(const PointCloud2::ConstSharedPtr inp
   pcl::fromROSMsg(*input_msg, *pointcloud_ptr);
 
   for (const auto & point : pointcloud_ptr->points) {
-    no_height_pointcloud_ptr->push_back(pcl::PointXYZ(point.x, point.y, 0.0));
+    no_height_pointcloud_ptr->push_back(pcl::PointXYZ(point.x, point.y, point.z));
```

Comment:
With this change, `*no_height_pointcloud_ptr` becomes a simple copy of `*pointcloud_ptr` so I think you can remove this loop and replace the `no_height_pointcloud_ptr` with `pointcloud_ptr`.

Reply:
@maxime-clem Thank you. I fix it.